### PR TITLE
User app

### PR DESCRIPTION
**Pull request by rremii**

### CODE REVIEW

File: src/shared/features/CreateEditBookingModal/ui/HostInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import { View, Text } from 'react-native'
+import { Avatar } from '@shared/ui/Avatar'
+import { Tag } from '@shared/ui/Tag'
+import { StyleSheet } from 'react-native'
+
+export const HostInfo = () => {
+  return (
+    <View style={{ alignItems: 'center', gap: 10 }}>
```
Comment (PetrifiedEye) [inline src/shared/features/CreateEditBookingModal/ui/HostInfo.tsx:8]: Inline styles, please move to StyleSheet

File: src/shared/moduls/datePicker/DatePicker.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+export const DatePicker = ({ initDate, onChange, ...props }: Props) => {
+  const [date, setDate] = useState<Date | null>(null)
+  const [isPicker, setIsPicker] = useState(false)
+
+  useEffect(() => {
+    setDate(initDate)
+  }, [initDate])
+
+  useEffect(() => {
```
Comment (PetrifiedEye) [inline src/shared/moduls/datePicker/DatePicker.tsx:31]: Looks like useEffect not needed here

File: src/shared/moduls/tagsPicker/TagsPicker.tsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    if (!onChange) return
+    onChange(tags.filter((t) => t !== tag))
+  }
+
+  const startAdding = () => {
+    setIsAdding(true)
+  }
+
+  useEffect(() => {
+    if (!inputRef.current) return
```
Comment (PetrifiedEye) [inline src/shared/moduls/tagsPicker/TagsPicker.tsx:44]: Check TextInput autofocus property

File: src/shared/ui/LabelWithEdit.tsx
Comment (PetrifiedEye) [inline src/shared/ui/LabelWithEdit.tsx:1]: Make this input uncontrolled

File: src/shared/ui/ProfileStatus.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+export const ProfileStatus = ({ status }: Props) => {
+  return (
+    <View style={styles.container}>
+      <Text style={styles.text}>Your status: {status}</Text>
+    </View>
+  )
+}
+
+const styles = StyleSheet.create({
+  container: {},
```
Comment (PetrifiedEye) [inline src/shared/ui/ProfileStatus.tsx:18]: Pls remove

**Review by yab154 — APPROVED** [review]

**Pull request merged**